packages/fxa-settings/src/components/Settings/ProgressBar/en.ftl

```diff
@@ -0,0 +1,7 @@
+## Progress bar
+
+# This is the aria-label text for the progress bar. The progress bar is meant to visually show the user how much progress they have made through the steps of a given flow.
+# Variables:
+#   $currentStep (number) - the step which the user is currently on
+#   $numberOfSteps (number) - the total number of steps in a given flow
+progress-bar-aria-label = A progress bar showing that the user is on step {$currentStep} of {$numberOfSteps}.
```

Comment:
I'm curious, is there a style for aria labels?
I see many of the strings use `aria-label` as an attribute, example:
https://github.com/mozilla/fxa/blob/def80f81d48ecbfbf54b7e0ef487a8a2c753771e/packages/fxa-settings/src/components/images/en.ftl

Reply:
I'll defer to wiser minds, but I think `aria-label` ([MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)) is only used for screen readers and is not meant to be part of the UI.
> _**Note:** The `aria-label` is only "visible" to assistive technologies. If the information is important enough to add for AT users, consider making it visible for all users._

Reply:
I've been confused about that label historically but since there is a precedent, I'm more than happy to add it!

Reply:
I'll leave the call on what to do up to those with more knowledge, just something I noticed but doesn't really affect localization.